arch/arm/core/cortex_m/Kconfig

```diff
@@ -203,7 +205,7 @@ config SW_VECTOR_RELAY
 	prompt "Enable Cortex-M0 Vector Table soft relay"
 	default n
 	default y if BOOTLOADER_MCUBOOT
-	depends on CPU_CORTEX_M0 && !CPU_CORTEX_M0_HAS_VECTOR_TABLE_REMAP
+	depends on (CPU_CORTEX_M0 && !CPU_CORTEX_M0_HAS_VECTOR_TABLE_REMAP) || (CPU_CORTEX_M0PLUS && !CPU_CORTEX_M_HAS_VTOR)
```

Comment:
Why CPU_CORTEX_M0_HAS_VECTOR_TABLE_REMAP is missing from the conditional of M0+?

Is it possible to have an M0+, no VTOR implemented, and no CPU_CORTEX_M0_HAS_VECTOR_TABLE_REMAP supported?

Reply:
agreed, addressed